src/WakuProvider.tsx

```diff
@@ -0,0 +1,134 @@
+import React from "react";
+import type { Waku } from "@waku/interfaces";
+
+import type {
+  BootstrapNodeOptions,
+  CrateWakuHook,
+  FullNodeOptions,
+  LightNodeOptions,
+  RelayNodeOptions,
+} from "./types";
+import {
+  useCreateFullNode,
```

Comment:
full node is for test convenience only, I would not include that in the React components because we don't want lib consumer to run full node in their browser

Reply:
Are there docs somewhere? I know @hackyguru mentioned preparing a cheat sheet but we did not secure hackathon prizes after all.

Reply:
We do need docs but I assume `jsdocs` that I provided are enough for the time being and for the proper docs itself I am wondering what a goal would be? Do we want to showcase how to use lib (updated web-chat example), what hooks are doing (jsdocs provided), some place to have all info in one place (I will include README).

Reply:
removed